src/Framework/MSBuildEventSource.cs

```diff
@@ -53,6 +53,12 @@ public void ApplyLazyItemOperationsStart(string itemType)
             WriteEvent(1, itemType);
         }
 
+        [Event(1, Keywords = Keywords.All)]
```

Comment:
All the events need to have a unique number, as far as I know. You will need to assign the available number to this event (and to move it, I suppose, to the end of the file).

Reply:
I forgot to remove it. I was trying if EventSource will throw exception during construction or later. Removed.